docs/getting-started/installing/installing-core-linux.md

```diff
@@ -2,119 +2,71 @@
 
 By [Zlatko Knezevic](https://github.com/blackdwarf)
 
-These instructions will lead you through acquiring the .NET Core DNX SDK via the [.NET Version Manager (DNVM)](https://github.com/aspnet/dnvm) and running a “Hello World” demo on Linux.
-
-.NET Core NuGet packages and the .NET Core DNX SDKs are available on the [ASP.NET ‘vnext’ myget feed](https://www.myget.org/F/aspnetvnext), which you can more easily view on [gallery](https://www.myget.org/gallery/aspnetvnext) for the feed.
+This document will lead you through acquiring the .NET Core and its associated CLI toolchainand running a “Hello World” demo on Linux.
 
 ## Setting up the environment
 
-These instructions have been written and tested on Ubuntu 14.04 LTS, since that is the main Linux distribution the .NET Core team uses. These instructions may succeed on other distributions as well. We are always accepting new pull requests on [our GitHub repo](https://www.github.com/dotnet/coreclr/) that address running on other Linux distributions. The only requirement is that they do not break the ability to use Ubuntu 14.04 LTS.
-
-### Installing the required packages
-
-Install the `libunwind8`, `libssl-dev` and `unzip` packages:
-
-```console
-sudo apt-get install libunwind8 libssl-dev unzip
-
-```
+These instructions have been written and tested on Ubuntu 14.04 LTS, since that is the main Linux distribution the .NET Core team uses.  We are always accepting new pull requests on [our GitHub repo](https://www.github.com/dotnet/coreclr/) that address running on other Linux distributions. The only requirement is that they do not break the ability to use Ubuntu 14.04 LTS.
 
-### Certificates
+### Setting up the apt-get feed
 
-You need to import trusted root certificates in order to restore NuGet packages. You can do that with the `mozroots` tool.
+We will use apt-get, the native Ubuntu package installer, to install the .NET Core SDK. In order to get the package, however, we will need to add a new package feed. The below commands will do this. 
```

Comment:
We should exemplify the experience for all supported OS platforms and not just Ubuntu.